CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.75 [2023-07-07]
+
+_Bug fixes_
+
+- Fixed the formatting in CIS v2.0.0 benchmark documents which would cause the control page to not render correctly on the hub. ([#687](https://github.com/turbot/steampipe-mod-aws-compliance/pull/687))
```

Comment:
```suggestion
- Fixed the formatting in CIS v2.0.0 benchmark documents which would cause several control pages to not render correctly on the hub. ([#687](https://github.com/turbot/steampipe-mod-aws-compliance/pull/687))
```